specs/crawler/common/parameters.yml

```
@@ -1,18 +1,18 @@
 CrawlerIdParameter:
   name: id
   in: path
-  description: The Id of the targeted Crawler.
+  description: Crawler ID.
   required: true
   schema:
-    type: string
+    $ref: '#/CrawlerId'
 
 TaskIdParameter:
-  name: tid
+  name: taskID
   in: path
-  description: The Id of the targeted Task.
+  description: Task ID.
   required: true
   schema:
-    type: string
+    $ref: '#/TaskId'
```

```suggestion
    $ref: '#/TaskID'
```

idk what other thinks but I'm team capital D and I think we do that already elsewhere